javascript/index.js

```diff
@@ -40,26 +56,63 @@ function clearSplits() {
 
 function setStopBtn() {
   // ... your code goes here
+  btnLeft.innerHTML = 'STOP'
+  btnLeft.className = 'btn stop'
 }
 
 function setSplitBtn() {
   // ... your code goes here
+  btnRight.innerText = 'SPLIT'
+  btnRight.className = 'btn split'
 }
 
 function setStartBtn() {
   // ... your code goes here
+  btnLeft.innerText = 'START'
+  btnLeft.className = 'btn start'
 }
 
 function setResetBtn() {
   // ... your code goes here
+  btnRight.innerText = 'RESET'
+  btnRight.className = 'btn reset'
 }
 
 // Start/Stop Button
 btnLeft.addEventListener('click', () => {
   // ... your code goes here
+  if(btnLeft.innerText === 'START'){
+    setStopBtn()
+    chronometer.startClick(printTime)
+    setSplitBtn()
+  } else {
+    setStartBtn()
+    setResetBtn()
+    chronometer.stopClick()
+  }
+
+  
+ 
+
+  //console.log('left button works')
+  //chronometer.startClick(printTime)
+  //secUni.innerHTML = chronometer.currentTime
 });
 
 // Reset/Split Button
 btnRight.addEventListener('click', () => {
   // ... your code goes here
+  if(btnRight.innerText === 'RESET'){
+    //setSplitBtn()
+    chronometer.startClick(printTime)
```

Comment:
when the button reads RESET you want to run the chronometer method for reset. not start the chronometer again.